ruma-client-api/src/r0/backup/get_backup_key_session.rs

```diff
@@ -0,0 +1,36 @@
+//! [GET /_matrix/client/r0/room_keys/keys/{roomId}/{sessionId}](https://matrix.org/docs/spec/client_server/unstable#get-matrix-client-r0-room-keys-keys-roomid-sessionid)
+
+use ruma_api::ruma_api;
+use ruma_identifiers::RoomId;
+
+ruma_api! {
+    metadata: {
+        description: "Retrieve a key from the backup",
+        method: GET,
+        name: "get_backup_key_session",
+        path: "/_matrix/client/r0/room_keys/keys/:room_id/:session_id",
+        rate_limited: true,
+        authentication: AccessToken,
+    }
+
+    request: {
+        /// The backup version. Must be the current backup.
+        #[ruma_api(query)]
+        pub version: String,
+
+        /// Room ID.
+        #[ruma_api(path)]
+        pub room_id: RoomId,
+        /// Session ID.
+        #[ruma_api(path)]
+        pub session_id: String,
+    }
+
+    response: {
+        /// Key data.
+        #[ruma_api(body)]
+        pub key_data: Option<super::KeyData>,
```

Comment:
Does `Option<...>` make sense here? [spec](https://matrix.org/docs/spec/client_server/unstable#get-matrix-client-r0-room-keys-keys-roomid-sessionid)

Reply:
No, I don't think this should be optional. If no key data is found, this will return `M_NOT_FOUND` / 404.